tensorboard/webapp/widgets/line_chart_v2/lib/worker/worker_chart_utils.ts

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {DataSeries} from '../internal_types';
+
+export interface CompactDataSeries {
+  idsAndLengths: Array<{id: string; length: number}>;
+  flattenedSeries: ArrayBufferLike;
```

Comment:
Why `ArrayBufferLike` rather than `ArrayBuffer`? Doesn’t it have to be
literally an `ArrayBuffer` to be `Transferable`?